fuuk/people/tests/test_admin.py

```diff
@@ -48,3 +55,263 @@ def test_change_view(self):
         response = self.client.get(reverse('admin:people_course_change', args=(course.pk, )))
 
         self.assertContains(response, 'Change course')
+
+
+@override_settings(LANGUAGE_CODE='en')
+class TestGrantAdmin(TestCase):
+    """
+    Test `GrantAdmin`.
+    """
+    def setUp(self):
+        self.superusername = 'superuser'
+        self.superpwd = 'superuser'
+        self.username1 = 'user'
+        self.pwd1 = 'user'
+        self.username2 = 'user2'
+        self.pwd2 = 'user2'
+        self.request = MockRequest()
```

Comment:
I used the same method as Django and I think, it satisfies. See Django source:
https://github.com/django/django/blob/master/tests/modeladmin/tests.py#L23-L32